core/views/unsorted.py

```diff
@@ -2301,15 +2301,17 @@ def process_unsaved_booking(request):
     return
 
 
-def user_login(request):
+def user_login(request, username=None):
```

Comment:
I would suggest that we use https://djangopackages.org/packages/p/django-registration/ for registration / login. If you agree let's open a new issue for that.

Reply:
Yes sounds good! These views looked like a bit of a mess but I didn’t want to refactor them now. 🙈

Reply:
There will come a day :)

Reply:
yes agree! this project was started before django had a registration or profile functionality built in. have been keen to move to the newer functionality.